CRM/Activity/BAO/Query.php

```diff
@@ -440,7 +442,7 @@ public static function from($name, $mode, $side) {
    * rather than a static function.
    */
   public static function getSearchFieldMetadata() {
-    $fields = ['activity_type_id'];
+    $fields = ['activity_type_id', 'activity_date_time'];
```

Comment:
this is the key change - adding the field to this array means it can be removed from other places